modules/ui_patterns_layouts/ui_patterns_layouts.api.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * @file
+ * API file.
+ */
+
+use Drupal\ui_patterns\Definition\PatternDefinition;
+
+/**
+ * Alter pattern layout "Manage display" form.
+ *
+ * @param array $form
+ *   Pattern settings fieldset.
+ * @param \Drupal\ui_patterns\Definition\PatternDefinition $definition
+ *   Pattern definition.
+ * @param array $configuration
+ *   Pattern configuration.
+ *
+ */
+function hook_ui_patterns_layout_form_alter(array &$form, PatternDefinition $definition, array $configuration) {
```

Comment:
The name of the module includes a trailing `s`, could you please rename the hook into:
```
hook_ui_patterns_layouts_display_settings_form_alter
```